public/reference/functions/tables/node.md

```diff
@@ -93,10 +93,10 @@ bikes
 Connect directly to DynamoDB.
 
 ```javascript
-let data = require('@architect/data')
+let arc = require('@architect/functions')
```

Comment:
is it preferred to use destructuring here?

Reply:
no preference!